### PR TITLE
fix: get container logs on console in e2es

### PR DESCRIPTION
**Pull request by JSPRH**
# Description

Provide the container logs to the console for failed e2e tests.

See [this GH Action run](https://github.com/kordis-leitstelle/kordis/actions/runs/8000326648/job/21849566332?pr=685#step:20:1) for the output of a failed run.

# Checklist:

- [x] The title of this PR and the commit history is conform with
	the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.
- [x] I have performed a self-review of my own code.
- [x] My changes generate no new warnings, SonarCloud reports no Vulnerabilities, Bugs or Code Smells.
- [x] I have added tests (unit and E2E if user-facing) that prove my fix is effective or that my feature works,
	Coverage > 80% and not less than the current coverage of the main branch.
- [x] The PR branch is up-to-date with the base branch. In case you merged `main` into your feature branch, make sure you have run the latest NX migrations (`nx migrate --run-migrations`).

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -63,10 +63,11 @@ jobs:
         run: ./tools/db/kordis-db.sh init e2edb
       - name: Start API and SPA containers
         run: |
-          docker run -d -p 3000:3333 -e MONGODB_URI=mongodb://host.docker.internal:27017/e2edb kordis-api:${{ github.sha }}
-          docker run -d -p 4200:8080 -e API_URL=http://localhost:3000 -e AUTH_PROVIDER=dev kordis-spa:${{ github.sha }}
+          docker run -d --name kordis-api-container -p 3000:3333 -e MONGODB_URI=mongodb://172.17.0.1:27017/e2edb -e AUTH_PROVIDER=dev kordis-api:${{ github.sha }}
+          docker run -d --name kordis-spa-container -p 4200:8080 -e API_URL=http://localhost:3000 -e AUTH_PROVIDER=dev kordis-spa:${{ github.sha }}
```
Comment (timonmasberg) [inline .github/workflows/ci.yml:67]: there is no `AUTH_PROVIDER` for spa, this was wrong in the first place

**Comment by timonmasberg** [comment]
very good changes

File: .github/workflows/ci.yml (hunk 1 above)
Comment (JSPRH) [reply .github/workflows/ci.yml]: Good catch! There it went 😂

**Review by timonmasberg — APPROVED** [review]

**Pull request merged**